doc/classes/CanvasItem.xml

```diff
@@ -339,7 +339,7 @@
 		<method name="get_canvas_transform" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the transform matrix of this item's canvas.
+				Returns the transform from the [Viewport]s coordinate system to the coordinate system of the canvas, this item is in.
```

Comment:
It transforms the other way around.

Reply:
Wow. I did not see my mistake there. Thanks. Will update the PR.